MONEY/NetworkEarnFromIslandHeistDjMission.md

```diff
@@ -0,0 +1,23 @@
+---
+ns: MONEY
+aliases: ["0xE2BB399D90942091", "_NETWORK_EARN_FROM_ISLAND_HEIST_DJMISSION"]
+---
+## NETWORK_EARN_FROM_ISLAND_HEIST_DJ_MISSION
+
+```c
+// 0xE2BB399D90942091
+void NETWORK_EARN_FROM_ISLAND_HEIST_DJ_MISSION(int amount, int mission);
+```
+
+Reward the player with money earned from completing DJ missions during the Island Heist event.
+
+```
+NativeDB Introduced: v2189
+```
+
+## Parameters
+* **amount**: The amount of money earned from the DJ mission.
+* **mission**: The identifier of the DJ mission completed.
+
+## Return value
```

Comment:
If the native doesn't return any value you don't need to include `## Return value`

Reply:
Oh didn't know, i'm going to update my PR thanks